### PR TITLE
Update src dir part-1

### PR DESCRIPTION
**Pull request by yaacov**
## 📝 Description

> Add a brief description

## 🎥 Demo

> Please add a video or an image of the behavior/changes

### CODE REVIEW

**Review by avivtur — COMMENTED** [review]
looking good I like this structure, just minor nitpicks to make this perfect :)

File: src/virtualmachines/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# VirtualMAchine
```
Comment (avivtur) [inline src/virtualmachines/README.md:1]: please change to `VirtualMachine` :)

File: src/virtualmachinesInstance/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# VirtualMAchineIntance
```
Comment (avivtur) [inline src/virtualmachinesInstance/README.md:1]: please change to `VirtualMachineInstance` :)
I also think we should change the directory name to `virtualmachineinstance` for naming consistency

File: src/templates/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Templates
+
+## Importasnt -
```
Comment (avivtur) [inline src/templates/README.md:3]: please change to `Important` :)

File: src/catalog/README.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# VirtualMachine Catalog
+
+Caltalog deisplay a tile view of all available templates
```
Comment (avivtur) [inline src/catalog/README.md:3]: please change to `Catalog display` :)

**Review by avivtur — COMMENTED** [review]
super small nitpicks to make this perfect

File: src/virtualmachines/README.md (hunk 1 above)
Comment (yaacov) [reply src/virtualmachines/README.md:1]: Thanks !

File: src/virtualmachinesInstance/README.md (hunk 2 above)
Comment (yaacov) [reply src/virtualmachinesInstance/README.md:1]: should be the kink ... will look what the kind is

File: src/templates/README.md (hunk 3 above)
Comment (yaacov) [reply src/templates/README.md:3]: :-)

File: src/catalog/README.md (hunk 4 above)
Comment (yaacov) [reply src/catalog/README.md:3]: +1

**Pull request closed without merging**